src/tdastro/sources/snia_host.py

```diff
@@ -0,0 +1,17 @@
+from tdastro.sources.physical_model import PhysicalModel
+
+class SNIaHost(PhysicalModel):
+    """A static source.
+
+    Attributes
+    ----------
+    radius_std : `float`
```

Comment:
The attributes can be deleted from this doc string since you don't use them in this class.